docs/docs/general/fonts.md

```diff
@@ -2,20 +2,22 @@
 title: Fonts
 ---
 
-There are 4 special global settings that can change all of the fonts on the chart. These options are in `Chart.defaults`. The global font settings only apply when more specific options are not included in the config.
+There is a special global settings that can change all of the fonts on the chart. These option is `Chart.defaults.font`. The global font settings only apply when more specific options are not included in the config.
```

Comment:
This is a mix of singular and plural. If there's just one option now it should be "There is a special global setting" and "This option is"

I think the old way was mostly fine too, but removing "4" was a good improvement

Either way should be fine really